src/shared/utils/exportData/parseResponseValue.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-case-declarations */
```

Comment:
u can use just { ... } for `case` and remove this line

```
case ItemResponseType.Time: {
....
}
```

Reply:
fixed